website/templates/website/error_pages/404.html

```diff
@@ -0,0 +1,9 @@
+{% extends "website/base_template.html" %}
+
+{% block content_body %}
+
+<!-- Content -->
+<div class="content-card">
+  <div class="page-heading"><h2>404 Not Found</h2></div>
```

Comment:
We might want to have a "cuter" error page, maybe with a tractography image or something... This could be added as an issue, to do in a later PR.